backend/src/main/java/ca/bc/gov/backendstartapi/enums/ActivityEnum.java

```diff
@@ -0,0 +1,55 @@
+package ca.bc.gov.backendstartapi.enums;
+
+import java.util.Optional;
+import lombok.Getter;
+import lombok.ToString;
+
+/** This enumeration contains all possible activities. */
+@Getter
+@ToString
+public enum ActivityEnum {
+  SEEDLOT_REGISTRATION(
+      "SoilMoistureField",
+      "Seedlot registration",
+      "Start a new registration or check on existing seedlots registrations",
+      "/seedlot-registration"),
+  PARENT_TREE_ORCHARD(
+      "Tree",
+      "Parent tree orchard",
+      "Manage the parent trees inside your orchard and look for their "
+          + "latest reports and the latest data.",
+      "/parent-tree-orchard"),
+  SEEDLING_REQUEST(
+      "CropGrowth",
+      "Seedling request",
+      "Open a new seedling request for your reforestation needs.",
+      "/seedling-request");
+
+  private final String iconName;
+  private final String title;
+  private final String description;
+  private final String page;
+
+  private ActivityEnum(
+      final String iconName, final String title, final String description, final String page) {
+    this.iconName = iconName;
+    this.title = title;
+    this.description = description;
+    this.page = page;
+  }
```

Comment:
@thiagosanchezenc and I discussed about this, and this enumeration exists because we need to store user's favorite activities for each page, we must have in the back-end a reference for the pages. Another key factor is that every time the user logs in, the app will need to load the menu according to his roles, showing only what that user can see. So we decided to go with what those page are, grabbing from the server.